be/src/exec/parquet_reader.cpp

```diff
@@ -47,6 +48,22 @@ ParquetReaderWrap::ParquetReaderWrap(std::shared_ptr<arrow::io::RandomAccessFile
     _properties.enable_buffered_stream();
     _properties.set_buffer_size(8 * 1024 * 1024);
     _filename = (reinterpret_cast<ParquetChunkFile*>(_parquet.get()))->filename();
+
+    const std::string unit_config = config::parquet_coerce_int96_timestamp_unit;
```

Comment:
If not set the parquet_coerce_int96_timestamp_unit, what's the behavior?

Reply:
The config is empty, and the unit would be default value `MICRO`.